phoonnx/model_manager.py

```
@@ -800,6 +800,132 @@ def get_piper_community_voice_list(self):
+            voice = TTSModelInfo(
+                voice_id="piper_" + f"dividebysandwich/en-US_{voice}",
+                lang="en-US",
+                model_url=url,
+                config_url=url + ".json",
+            )
+            self.add_voice(voice)
+
+        # https://huggingface.co/russdill/kronk
+        url = "https://huggingface.co/russdill/kronk/resolve/main/en/en_US/kronk/medium/kronk-medium.onnx"
+        voice = TTSModelInfo(
+            voice_id="piper_" + f"russdill/en-US_kronk",
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)